ChangeLog.md

```diff
@@ -63,6 +69,10 @@
 
   * Fix `pl.inner_html(...)` helper function (Nathan Walters).
 
+  * Fix slow gradebook API by removing `last_submission_date` (Matt West).
+
+  * Fix API IDs to be JSON integers (Matt West).
```

Comment:
The main issue I ran into was that we were sometimes returning them as integers and sometimes as strings. I don’t especially care which one we use, but I wanted it to be consistent. This integer/string discrepency happens because `json_build_object()` in postgres converts bigints to JSON numbers, whereas `node-postgres` converts them to strings (to avoid overflow). I couldn’t see any way to avoid `json_build_object()` because we need to do at least some object construction inside the query, so I converted it to do all JSON conversions inside postgres, giving us consistent output.